.github/workflows/deploy-lmm.yaml

```diff
@@ -0,0 +1,36 @@
+# This workflow runs on pushes when a pull request is opened under certain branch conventions.
+
+name: Build and deploy dev sandbox
+
+on:
+  pull-request:
+    paths-ignore:
+      - 'docs/**'
+
+    branches:
+      - 'lmm/**'     
```

Comment:
I'm not quite sure this actually works but might have to open a test branch/PR after this merges to find out?

Reply:
> `pull-request` is not a valid event name

I'll work on fixing this up

Reply:
Fixed the error but still don't think it will run until after merging. I also plan to add something to the action to comment upon the pull requests in a follow-up.